add vscode mentions (#3898)
kedro-org/kedro: docs/source/get_started/install.md

```bash
conda deactivate
```

> ### Optional: Setup Kedro with IDE

Reviewer: ```suggestion
### Optional: Add Kedro support to your IDE
```

? (Unusure about this one as well, requesting @stichbury's opinion)

Author: > [Integrate ESLint in VSCode with the official extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)

Integrate Kedro in VS Code with the official extension? @astrojuanlu

Reviewer: +1!

> Working in IDEs can be a great productivity boost.

Reviewer: ```suggestion
Working in an IDE can be a great productivity boost.
```

> For VS Code Users: Checkout [Set up Visual Studio Code](../development/set_up_vscode.md) and [Kedro VS Code Extension](../development/set_up_vscode.md#kedro-vscode-extension)

Reviewer: I guess

```suggestion
For VS Code Users: Checkout [Set up Visual Studio Code](../development/set_up_vscode.md) and [Kedro VS Code Extension](../development/set_up_vscode.md#kedro-vs-code-extension)
```

?
